externs/d3.js

```diff
@@ -1,877 +1,995 @@
-d3.time.wednesdayOfYear.prototype = function () {};
-d3.time.thursday.prototype = function () {};
-d3.time.thursdays.prototype = function () {};
-d3.time.thursdayOfYear.prototype = function () {};
-d3.time.friday.prototype = function () {};
-d3.time.fridays.prototype = function () {};
-d3.time.fridayOfYear.prototype = function () {};
-d3.time.saturday.prototype = function () {};
-d3.time.saturdays.prototype = function () {};
-d3.time.saturdayOfYear.prototype = function () {};
-d3.time.week.prototype = function () {};
-d3.time.weeks.prototype = function () {};
-d3.time.weekOfYear.prototype = function () {};
-d3.time.month.prototype = function () {};
-d3.time.months.prototype = function () {};
-d3.time.year.prototype = function () {};
-d3.time.years.prototype = function () {};
-d3.time.scale.prototype = function () {};
-
-//
-// d3_externs.js ends here
+d3.transition.prototype = {
+  "constructor": function () {},
+  "select": function () {},
+  "selectAll": function () {},
+  "filter": function () {},
+  "merge": function () {},
+  "selection": function () {},
+  "transition": function () {},
+  "call": function () {},
+  "nodes": function () {},
+  "node": function () {},
+  "size": function () {},
+  "empty": function () {},
+  "each": function () {},
+  "on": function () {},
+  "attr": function () {},
+  "attrTween": function () {},
+  "style": function () {},
+  "styleTween": function () {},
+  "text": function () {},
+  "remove": function () {},
+  "tween": function () {},
+  "delay": function () {},
+  "duration": function () {},
+  "ease": function () {}
+};
+d3.hierarchy.prototype = {
+  "constructor": function () {},
+  "each": function () {},
+  "eachAfter": function () {},
+  "eachBefore": function () {},
+  "sum": function () {},
+  "sort": function () {},
+  "path": function () {},
+  "ancestors": function () {},
+  "descendants": function () {},
+  "leaves": function () {},
+  "links": function () {},
+  "copy": function () {}
+};
+d3.zoomIdentity.constructor.prototype = {
+  "constructor": function () {},
+  "scale": function () {},
+  "translate": function () {},
+  "apply": function () {},
+  "applyX": function () {},
+  "applyY": function () {},
+  "invert": function () {},
+  "invertX": function () {},
+  "invertY": function () {},
+  "rescaleX": function () {},
+  "rescaleY": function () {},
+  "toString": function () {}
+};
+d3.zoomTransform.prototype = {
+  "constructor": function () {},
+  "scale": function () {},
+  "translate": function () {},
+  "apply": function () {},
+  "applyX": function () {},
+  "applyY": function () {},
+  "invert": function () {},
+  "invertX": function () {},
+  "invertY": function () {},
+  "rescaleX": function () {},
+  "rescaleY": function () {},
+  "toString": function () {}
+};
+/**********************************************************************
+ * End Generated Extern for d3
+/**********************************************************************/
```

Comment:
(Line 3: ) `Generated by http://jmmk.github.io/javascript-externs-generator  ` The only "source" that I've found...